code/game/jobs/job/civilian.dm

```diff
@@ -46,7 +46,7 @@
 	spawn_positions = 2
 	supervisors = "the Chief Executive Officer"
 	difficulty = "Easy."
-	alt_titles = list("Culinary Artist","Cook")
+	alt_titles = list("Culinary Artist","Cook", "Line Chef", "Head Chef")
```

Comment:
```suggestion
	alt_titles = list("Culinary Artist","Cook", "Line Chef")
```
Head Chef seems a bit much for just a rando cook that there can be multiple of. I believe there's 2 slots for cooks, having 2 use 'Head Chef' would be strange.